primitives/rpc/src/lib.rs

```diff
@@ -178,17 +178,20 @@ sp_api::decl_runtime_apis! {
 	}
 }
 
+/// Fallback transaction converter when the `ConvertTransactionRuntimeApi` is not available. For almost all
+/// non-legacy cases, you can instantiate this type as `NoTransactionConverter`.
 pub trait ConvertTransaction<E> {
 	fn convert_transaction(&self, transaction: ethereum::TransactionV2) -> E;
 }
 
+/// No fallback transaction converter is available.
 // `NoTransactionConverter` is a non-instantiable type (an enum with no variants),
 // so we are guaranteed at compile time that `NoTransactionConverter` can never be instantiated.
 pub enum NoTransactionConverter {}
```

Comment:
Can we replace the `TransactionConverter` with `NoTransactionConverter` in the template node runtime?

The runtime has implemented the `ConvertTransactionRuntimeApi`, so there is no need for a fallback function there.